--- git-prompt.sh	2024-11-04 22:12:35.190093315 -0500
+++ git-prompt.sh.copy	2024-11-04 22:14:01.940092923 -0500
@@ -42,7 +42,9 @@
 # name.  You can configure this per-repository with the
 # bash.showDirtyState variable, which defaults to true once
 # GIT_PS1_SHOWDIRTYSTATE is enabled.
-#
+
+GIT_PS1_SHOWDIRTYSTATE=1
+
 # You can also see if currently something is stashed, by setting
 # GIT_PS1_SHOWSTASHSTATE to a nonempty value. If something is stashed,
 # then a '$' will be shown next to the branch name.
@@ -53,7 +55,9 @@
 # configure this per-repository with the bash.showUntrackedFiles
 # variable, which defaults to true once GIT_PS1_SHOWUNTRACKEDFILES is
 # enabled.
-#
+
+GIT_PS1_SHOWUNTRACKEDFILES=1
+
 # If you would like to see the difference between HEAD and its upstream,
 # set GIT_PS1_SHOWUPSTREAM="auto".  A "<" indicates you are behind, ">"
 # indicates you are ahead, "<>" indicates you have diverged and "="
@@ -72,7 +76,9 @@
 # find one, or @{upstream} otherwise.  Once you have set
 # GIT_PS1_SHOWUPSTREAM, you can override it on a per-repository basis by
 # setting the bash.showUpstream config variable.
-#
+
+GIT_PS1_SHOWUPSTREAM="verbose"
+
 # You can change the separator between the branch name and the above
 # state symbols by setting GIT_PS1_STATESEPARATOR. The default separator
 # is SP.
@@ -89,7 +95,9 @@
 # If you would like to see a notification on the prompt when there are
 # unresolved conflicts, set GIT_PS1_SHOWCONFLICTSTATE to "yes". The
 # prompt will include "|CONFLICT".
-#
+
+GIT_PS1_SHOWCONFLICTSTATE="yes"
+
 # If you would like to see more information about the identity of
 # commits checked out as a detached HEAD, set GIT_PS1_DESCRIBE_STYLE
 # to one of these values:
@@ -103,7 +111,9 @@
 # If you would like a colored hint about the current dirty state, set
 # GIT_PS1_SHOWCOLORHINTS to a nonempty value. The colors are based on
 # the colored output of "git status -sb".
-#
+
+GIT_PS1_SHOWCOLORHINTS=1
+
 # If you would like __git_ps1 to do nothing in the case when the current
 # directory is set up to be ignored by git, then set
 # GIT_PS1_HIDE_IF_PWD_IGNORED to a nonempty value. Override this on the
